src/wallet/walletdb.h

```diff
@@ -59,6 +63,11 @@ class CHDChain
         READWRITE(this->nVersion);
         READWRITE(nExternalChainCounter);
         READWRITE(masterKeyID);
+        if (this->nVersion >= SUPPORT_WATCHONLY_VERSION) {
+            READWRITE(isWatchOnly);
+            if(isWatchOnly)
+                READWRITE(masterPubKey);
```

Comment:
Somehow this PR was crashing here because of 
```
Assertion failed: (pubkey.size() == 33), function Encode, file pubkey.cpp, line 255.
```

It looks like that my tested `tpub` was non compressed? How is that even possible?
LLDB told me `pubkey = (vch = unsigned char [65] @ 0x00007fd16736976c)`.